### DISCUSSION TITLE
Version on Pip

### DISCUSSION THREAD

**Discussion by WilliamParsonsV** [CCSDSPy/ccsdspy#68] [Q&A] [answered]
Hi, me again! I'm just curious if you have the most up-to-date version on pip? I'm trying to integrate some converters but I'm getting import errors because my __init__.py doesn't have the line "from . import converters" that I can see is in the source code. I've done a pip install ccsdspy --upgrade but that doesn't seem to change the init file

I get the following error when trying to add that line manually:

File "c:\DevSoftware\Anaconda38\lib\site-packages\ccsdspy\__init__.py", line 9, in <module>
    from . import converters
ImportError: cannot import name 'converters' from partially initialized module 'ccsdspy' (most likely due to a circular import) (c:\DevSoftware\Anaconda38\lib\site-packages\ccsdspy\__init__.py)

**Comment by ddasilva** [comment]
The converters hadn't been released yet, but I think it's time for a release! I just uploaded something to PyPI, and it should work now.

**Reply by WilliamParsonsV** [reply]
Thanks! I copied the source code from github into my sitepackage folder manually and that worked as well, but I'll update properly now

**Reply by ddasilva** [reply, marked as answer]
No problem! You can also do this with pip `pip install git+https://github.com/CCSDSPy/ccsdspy`, and I think it should work as an entry in `environment.yml`

**Reply by WilliamParsonsV** [reply]
Okay good to know! My company uses gitlab so I wasn't sure how to properly do it on github but if pip can do it then that's even better
